docs/geedocs/5.3.2/answer/176738.html

```diff
@@ -309,6 +309,10 @@ <h2 id="status_request_timeout">Status request timeout</h2>
    spend waiting to fulfill a status request before it times out. This timeout can be configured manually by setting
    <code>MutexTimedWaitSec</code> in <code>/gevol/assets/.config/misc.xml</code></p>
 <code>&lt;MutexTimedWaitSec&gt;60&lt;/MutexTimedWaitSec&gt;</code>
+<br><br>
+
+<h2 id="improving_performance_in_resume_operations">Improving performance in resume operations</h2>
```

Comment:
I just realized the title specifically mentions resume. That should be changed to be more generic like the body text. Something like "Improving Operation Performance"